### ISSUE TITLE
skipfish segfault at crawler.c on ./skipfish -LVY -W dictionaries/complete.wl ...

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tinselcity/skipfish#122]
```
When I tried to run skipfish as
./skipfish -LVY -W dictionaries/complete.wl ... ,

skipfish crashed at crawler.c +645:

if (!strcasecmp((char*)lpos + 1, (char*)ex)){ i++; ck_free(tmp); continue; }

as lpos being NULL.

Quick workaround attached. Please check. Thanks in advance.

```

Original issue reported on code.google.com by `SangHoon...@gmail.com` on 6 Jul 2011 at 2:38

**Comment by GoogleCodeExporter** [comment]
```
Sorry, I forgot to write version info. the version was skipfish-2.01b.
And the patch file was corrupted. Attaching again.

```

Original comment by `SangHoon...@gmail.com` on 6 Jul 2011 at 2:41

Attachments:
- [diffs.txt](https://storage.googleapis.com/google-code-attachments/skipfish/issue-122/comment-1/diffs.txt)

**Comment by GoogleCodeExporter** [comment]
```
Funny, good catch. Will release a fixed version tomorrow.
```

Original comment by `lcam...@gmail.com` on 6 Jul 2011 at 2:57
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Oops, forgot to close the bug. This is fixed with 2.02. Thanks for the report.
```

Original comment by `lcam...@gmail.com` on 10 Jul 2011 at 7:05
- Changed state: **Fixed**
